<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="SketchPaper" name="sketch_paper" native="true"/>
    </item>
    <item>
     <widget class="SketchPaperSetter" name="sketch_paper_setter" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_showme"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save">
   <property name="text">
    <string>저장</string>
   </property>
  </action>
  <action name="action_showme">
   <property name="text">
    <string>제출자</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>종료</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SketchPaper</class>
   <extends>QWidget</extends>
   <header>sketch_paper.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SketchPaperSetter</class>
   <extends>QWidget</extends>
   <header>sketch_paper_setter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
